### ISSUE TITLE
0.26.0 installs meta_mpd.py with executable permissions only

### ISSUE THREAD

**Issue by yurivict** [badaix/snapcast#1021]
```
$ ls -l ./work/stage/usr/local/share/snapserver/plug-ins/meta_mpd.py
---x--x--x  1 yuri  wheel  33278 Dec 22 09:40 ./work/stage/usr/local/share/snapserver/plug-ins/meta_mpd.py
```

**Comment by badaix** [comment]
Could you be a bit more specific? 
What OS? 
How did you install it?

**Comment by yurivict** [comment]
OS: FreeBSD 13.1

Installed using the FreeBSD ports framework. It just uses the 'install' target.

**Comment by badaix** [comment]
Cmake install or make install?

**Comment by yurivict** [comment]
cmake

**Comment by badaix** [comment]
duplicate of #970

**Comment by badaix** [comment]
fixed in 7ab3daa76d9bd2700b609162c6cbe47351e535d0

**Comment by badaix** [comment]
Fixed in [snapcast v0.27.0](https://github.com/badaix/snapcast/releases/tag/v0.27.0)
